Assets/MixedRealityToolkit.SDK/Features/Utilities/Solvers/HandConstraintPalmUp.cs

```diff
@@ -15,17 +16,18 @@ public class HandConstraintPalmUp : HandConstraint
     {
         [Header("Palm Up")]
         [SerializeField]
+        [FormerlySerializedAs("facingThreshold")]
         [Tooltip("The angle (in degrees) of the cone between the palm's up and camera's forward have to match. Only supported by IMixedRealityHand controllers.")]
         [Range(0.0f, 90.0f)]
-        private float facingThreshold = 80.0f;
+        private float facingCameraTrackingThreshold = 80.0f;
 
         /// <summary>
         /// The angle (in degrees) of the cone between the palm's up and camera's forward have to match. Only supported by <see cref="Microsoft.MixedReality.Toolkit.Input.IMixedRealityHand"/> controllers.
         /// </summary>
-        public float FacingThreshold
+        public float FacingCameraTrackingThreshold
         {
-            get { return facingThreshold; }
-            set { facingThreshold = value; }
+            get { return facingCameraTrackingThreshold; }
+            set { facingCameraTrackingThreshold = value; }
```

Comment:
nit: while here can you switch to using =>
i.e
```
get => facingCameraTrackingThreshold;
set => facingCameraTrackingThreshold = value;
```

Reply:
I can but none of the other props in the class use this.

Reply:
sorry I meant for all of the props